addons/rose/addon/components/rose/table/row/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable ember/no-classic-components, ember/no-classic-classes  */
```

Comment:
Should we have some of these be handled in `.template-lintrc.js`? That way we could turn off some of these rules for multiple files in one place and they're centralized so we can easily revisit if the rules can be turned on again in the future.

Thoughts?